03.bowling/bowl.rb

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
```

Comment:
若干高等テクニックですが、flat_mapメソッドを活用できます。（flat_mapメソッドについて調べてみましょう）

```suggestion
shots = scores.flat_map do |s|
  s == 'X' ? [10, 0] : s.to_i
end
```